### PR TITLE
[T4A7][W09-B2]Yang Suwei

### PR DESCRIPTION
**Pull request by swxsw**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -60,5 +60,11 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
+    
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if ...
+     */
+     public boolean isSimilar(Name other) {}
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Name.java]: Good try, but you are a bit off from TDD. For example, a TDD session starts by adding a failing test, not by adding an empty method. Please take a look at the course handouts and suggested readings given in the assignment description to get a good understanding of TDD.

**Comment by damithc** [comment]
Some comments added. Please close the PR after reading comments.
